### ISSUE TITLE
unknown op: BestSplits (windows 10 + python 3.5.3)

### ISSUE THREAD

**Issue by netroby** [tensorflow/tensorflow#8644]
After install tensorflow on windows 10 with python 3.5.3, I try to run hello world. but got some message

```bash
PS C:\Users\huzhifeng> python --version
Python 3.5.3
PS C:\Users\huzhifeng> python -m pip  install -U tensorflow
Collecting tensorflow
  Downloading tensorflow-1.0.1-cp35-cp35m-win_amd64.whl (14.7MB)
    100% |████████████████████████████████| 14.7MB 40kB/s
Collecting protobuf>=3.1.0 (from tensorflow)
  Downloading protobuf-3.2.0-py2.py3-none-any.whl (360kB)
    100% |████████████████████████████████| 368kB 73kB/s
Collecting numpy>=1.11.0 (from tensorflow)
  Downloading numpy-1.12.1-cp35-none-win_amd64.whl (7.7MB)
    100% |████████████████████████████████| 7.7MB 61kB/s
Collecting six>=1.10.0 (from tensorflow)
  Using cached six-1.10.0-py2.py3-none-any.whl
Collecting wheel>=0.26 (from tensorflow)
  Downloading wheel-0.29.0-py2.py3-none-any.whl (66kB)
    100% |████████████████████████████████| 71kB 46kB/s
Collecting setuptools (from protobuf>=3.1.0->tensorflow)
  Downloading setuptools-34.3.2-py2.py3-none-any.whl (389kB)
    100% |████████████████████████████████| 399kB 119kB/s
Collecting appdirs>=1.4.0 (from setuptools->protobuf>=3.1.0->tensorflow)
  Downloading appdirs-1.4.3-py2.py3-none-any.whl
Collecting packaging>=16.8 (from setuptools->protobuf>=3.1.0->tensorflow)
  Downloading packaging-16.8-py2.py3-none-any.whl
Collecting pyparsing (from packaging>=16.8->setuptools->protobuf>=3.1.0->tensorflow)
  Downloading pyparsing-2.2.0-py2.py3-none-any.whl (56kB)
    100% |████████████████████████████████| 61kB 93kB/s
Installing collected packages: six, appdirs, pyparsing, packaging, setuptools, protobuf, numpy, wheel, tensorflow
  Found existing installation: setuptools 28.8.0
    Uninstalling setuptools-28.8.0:
      Successfully uninstalled setuptools-28.8.0
Successfully installed appdirs-1.4.3 numpy-1.12.1 packaging-16.8 protobuf-3.2.0 pyparsing-2.2.0 setuptools-34.3.2 six-1.
10.0 tensorflow-1.0.1 wheel-0.29.0
PS C:\Users\huzhifeng> tensorflow
PS C:\Users\huzhifeng> python
Python 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 16:02:32) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
>>> hello = tf.constant('Hello, TensorFlow!');
>>> sess = tf.Session();
>>> print(sess.run(hello));
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "BestSplits" device_type: "CPU"') for unknown op: BestSplits
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "CountExtremelyRandomStats" device_type: "CPU"') for unknown op: CountExtremelyRandomStats
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "FinishedNodes" device_type: "CPU"') for unknown op: FinishedNodes
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "GrowTree" device_type: "CPU"') for unknown op: GrowTree
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "ReinterpretStringToFloat" device_type: "CPU"') for unknown op: ReinterpretStringToFloat
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "SampleInputs" device_type: "CPU"') for unknown op: SampleInputs
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "ScatterAddNdim" device_type: "CPU"') for unknown op: ScatterAddNdim
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "TopNInsert" device_type: "CPU"') for unknown op: TopNInsert
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "TopNRemove" device_type: "CPU"') for unknown op: TopNRemove
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "TreePredictions" device_type: "CPU"') for unknown op: TreePredictions
E c:\tf_jenkins\home\workspace\release-win\device\cpu\os\windows\tensorflow\core\framework\op_kernel.cc:943] OpKernel ('
op: "UpdateFertileSlots" device_type: "CPU"') for unknown op: UpdateFertileSlots
b'Hello, TensorFlow!'
>>> print(sess.run(hello));
b'Hello, TensorFlow!'
>>> print(sess.run(hello));
b'Hello, TensorFlow!'
```

**Comment by drpngx** [comment]
/cc: @guschmue @mrry in case they know if these ops are in windows and why we would be getting these messages.

**Comment by guschmue** [comment]
I think this happens only in the 1.0 branch, master is fine. Maybe some commit missing into the 1.0 branch? I can take a look tomorrow unless @mrry knows.

**Comment by drpngx** [comment]
@netroby Can you try 1.0.1?

**Comment by guschmue** [comment]
had the same thought and just tried ... message is still there in 1.0.1

**Comment by drpngx** [comment]
@gunan @yifeif I don't know which commit that would be, and if it would be worth a cherry pick.

**Comment by gunan** [comment]
I think this was resolved 1 day after 1.0 branch cut.
You can try nightlies, or you can wait for 1.1

**Comment by mrry** [comment]
Yes, the `BestSplits` kernel has become [infamous](http://stackoverflow.com/q/42217532/3574081). Closing as resolved/a duplicate.
